ui/src/components/yaml/styles.module.less

```diff
@@ -223,7 +221,9 @@
       justify-content: space-between;
       align-items: center;
       padding: 12px;
```

Comment:
The use of percentages in the gradient (0% and 100%) is unnecessary and could be omitted for cleaner code.